deploy/helms/airflow/values.yaml

```diff
@@ -76,6 +76,11 @@ smtp:
   smtp_password: airflow
   smtp_mail_from: airflow@airflow.local
 
+slack:
```

Comment:
Maybe it should be moved to secrets?

Reply:
The config itself is a secret. The idea is that you may set this variable externally when you create a helm chart. You definitely don't have to store that data in git repo.
In fact, I don't think that moving the data for slack to a separate config file (a secret) is a good idea.